CHANGELOG.md

```diff
@@ -8,10 +8,11 @@
 ### Features
 
 ### Improvements
-- [\#607](https://github.com/Manta-Network/Manta/pull/607) Turn node client code into library for CLI project
 - [\#449](https://github.com/Manta-Network/Manta/pull/449) Remove strip from CI, and add strip profile to production.
 - [\#576](https://github.com/Manta-Network/Manta/pull/576) Unfilter xtokens.transfer_multicurrencies and bump MaxAssetsForTransfer to 2.
-- [\#609](https://github.com/Manta-Network/Manta/pull/609) Update parameter path from `sdk` to `manta-parameters`
```

Comment:
Found a mistake with the log. Seems i cant add commits to this branch for some reason so i have to leave it here like this. The 609 entry should be 

- [\#622](https://github.com/Manta-Network/Manta/pull/622) Update parameter path from `sdk` to `manta-parameters`.

Reply:
Updated